CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Changed
 
+- (core): Support Electron renderer process in js-runtime detector [#1348](https://github.com/bugsnag/bugsnag-js/pull/1348)
```

Comment:
Yeah I was on the fence about this. I know some teams are using the current notifier in the electron renderer, but of course it is not officially supported.

Reply:
> I know some teams are using the current notifier in the electron renderer

Would this change update their error types? I'm not sure what (if any) effect that would have in the mapping process right now.

Reply:
Actually I'm going to renege on that – if they're using the browser notifier it will be prebundled and have the static value `browserjs`, however if they are using the node notifier in the main process then it would get the new value `electronnodejs`